src/header/header.scss

```diff
@@ -265,4 +292,51 @@
       }
     }
   }
+
+  .iui-button-split-menu {
+    height: 100%;
+    padding: 0 0 0 $iui-xs;
+    display: flex;
+    overflow: hidden;
+    max-width: 25vw;
+
+    .iui-button {
+      padding: 0 $iui-xs;
+    }
+
+    &.iui-active {
+      // Basic active styling - blue background and bottom border
+      @include themed {
+        box-shadow: 0 2px 0 0 t(iui-color-foreground-primary);
+      }
+
+      &,
+      &:hover {
```

Comment:
So active hover state does not change background?

Reply:
With current implementation, none of active header buttons change on hover
![hoveraction](https://user-images.githubusercontent.com/83585998/128359571-7bae489f-a957-43a8-a10a-d7706dd7a88b.gif)

Reply:
Makes sense to preserve existing behavior I suppose.

I think when we decide what to do with active split button, we should also decide if this hover state is what we want.
@FlyersPh9

Reply:
How you have it working in the animation up above is good & what I was expecting. 😊

Reply:
So it's valid to have an active split button in the first place?